entities/src/increase_balance_receipt.rs

```diff
@@ -0,0 +1,22 @@
+//! `SeaORM` Entity. Generated by sea-orm-codegen 0.10.6
+
+use sea_orm::entity::prelude::*;
+use serde::{Deserialize, Serialize};
+
+#[derive(Clone, Debug, PartialEq, DeriveEntityModel, Eq, Serialize, Deserialize)]
+#[sea_orm(table_name = "increase_balance_receipt")]
+pub struct Model {
+    #[sea_orm(primary_key)]
+    pub id: i32,
+    #[sea_orm(unique)]
+    pub tx_hash: String,
```

Comment:
should we have this be an Option and then we can use the table for admin and referral credits, too?

Reply:
i think mixing up the table with referral credits would probably make it too convoluted. i thought about making tx-hash optional and allowing admin credits, but i dont see why we shouldnt just add another table, it will make the data more consistent (and disallow double-spending on the database layer in case any bugs make their way in the future). your call, I can also change it, but i think it'd prefer keeping three separate tables.